templates/default/contentlayer.config.js

```diff
@@ -6,11 +6,11 @@ import wikiLinkPlugin from "remark-wiki-link-plus";
 import remarkMath from "remark-math";
 import rehypeMathjax from 'rehype-mathjax';
 import codeExtra from "remark-code-extra";
-import mermaidPlugin from "mdx-mermaid";
```

Comment:
Why have we removed this?

Reply:
We are using mdx-mermaid component client side. This import was not being used.